src/exercise/05.js

```diff
@@ -4,24 +4,16 @@
 import * as React from 'react'
 import '../box-styles.css'
 
-// 💰 Use the className for the size and style (backgroundColor) for the color
-// 💰 each of the elements should also have the "box" className applied
-
-// 🐨 add a className prop to each of these and apply the correct class names
-// 💰 Here are the available class names: box, box--large, box--medium, box--small
-
-// 🐨 add a style prop to each of them as well so their background color
-// matches what the text says it should be as well as `fontStyle: 'italic'`
-const smallBox = <div>small lightblue box</div>
-const mediumBox = <div>medium pink box</div>
-const largeBox = <div>large orange box</div>
+function Box(props) {
+  return <div className={"box box--" + props.boxSize} style={{backgroundColor: props.boxColor}}>{props.boxText}</div>
+}
 
 function App() {
   return (
     <div>
-      {smallBox}
-      {mediumBox}
-      {largeBox}
+      <Box boxSize="small" boxColor="lightblue" boxText="small lightblue box"></Box>
+      <Box boxSize="medium" boxColor="pink" boxText="medium pink box"></Box>
+      <Box boxSize="large" boxColor="orange" boxText="large orange box"></Box>
```

Comment:
I could be wrong but I don't think this `boxText` approach is a normal pattern for React. If you do 

```
<Box>
  Hey!
</Box>
```

Then `'Hey!` gets passed in as the `children` prop of `props`, like this:

![image](https://user-images.githubusercontent.com/7775971/124175785-06cf9780-da7c-11eb-9cfb-cc8faeb7f1d4.png)

If you do

```
<Box>
  <span>Hey!</span>
  <span>Ho!</span>
</ Box>
```
Then the `children` prop of `props` will be an array with those `span` nodes, like so:

![image](https://user-images.githubusercontent.com/7775971/124175519-b35d4980-da7b-11eb-890f-6867656f9796.png)

Again, I'm not sure if your solution is, like, anti-react, but I don't think I've seen it that way.

Reply:
Also, it doesn't meet the AC 😆 

![image](https://user-images.githubusercontent.com/7775971/124176246-a725bc00-da7c-11eb-9325-d7fcb8befe68.png)